catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -450,4 +451,8 @@ public static int getBit(int number, int index) {
 		}
 		return 0;
 	}
+
+	public static String getStringfromResource(int resourceId) {
+		return CatroidApplication.getAppContext().getString(resourceId);
```

Comment:
accesssing context this way should be avoided.

this "getAppContext" method was a hack in the first place, put in place years ago, and we tried to remove it multiple times already.
please do not add further usages.

please find another way to put the string into its place, without accessing context from classes that did not need a context up to this point.

Reply:
Hey @84n4n4 , it seems that in order to access local application resources, android only provides the unique Resources class, which can be accessed only through the Context class. Under the hood of the resources class, it seems to use AssetManager (provided by the android system), but I am unable to get an instance of this class to get the string. Should I attempt to find a workaround with this AssetManager class?